src/strategies/index.ts

```diff
@@ -544,7 +545,11 @@ const strategies = {
   potion,
   'safety-module-bpt-power': safetyModuleBptPower,
   'minotaur-money': MinotaurMoney,
+<<<<<<< HEAD
+  'conv-finance': convFinance
+=======
   'sd-boost': sdBoost
+>>>>>>> bae574f0d0e4f06b513f7cf251c9b1531d4ed4f6
```

Comment:
Hi @eric-convergence please resolve this

Reply:
Didn't notice this conflict, thanks.